core/spec/models/spree/shipment_spec.rb

```diff
@@ -328,17 +329,17 @@
     end
   end
 
-  context "#cancel" do
-    it 'cancels the shipment' do
+  describe "cancel" do
```

Comment:
@alexblackie  it would probably be cleared if it was described as something like 'transition to cancel' or something of the ilk.